api/HTMLMediaElement.json

```diff
@@ -1338,10 +1338,10 @@
           "spec_url": "https://html.spec.whatwg.org/multipage/media.html#event-media-error",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "10"
```

Comment:
I doubt this is correct. The `MediaError` data says Chrome 1, and even though having `mediaElement.error` (which is a `MediaError`) without the "error" event is possible, I'm pretty sure they went together from the very start.